content/cli/commands/profile.md

```diff
@@ -2,3 +2,161 @@
 title: Profile
 weight: 50
 ---
+
+Fluvio provides support for multiple profiles out of the box, meaning that you can
+communicate with distinct clusters in an isolated and managed way. Each profile
+defines the means to communicate with a particular Fluvio cluster. One profile
+is active (or "current") at any given time, and all other Fluvio commands will
+interact with the cluster defined by the active profile. Profiles are defined
+in the Fluvio configuration file at `~/.fluvio/config`, and the `fluvio profile`
+family of commands is used to view and manipulate profiles.
+
+## `fluvio profile current`
+
+Prints out the name of the active Fluvio profile. Let's say you have a Fluvio Cloud account
+whose profile name is "cloud", and you have a local Fluvio cluster installed that
+you use for development, whose profile name is "local".
+
+If your current profile is "local", then a command such as [`fluvio topic create greeting`] will
+create a "greeting" topic on your local cluster.
+
+[`fluvio topic create greeting`]: /cli/commands/topic#fluvio-topic-create
+
+```
+fluvio-profile-current
+Print the name of the current context
+
+USAGE:
+fluvio profile current
+
+FLAGS:
+-h, --help    Prints help information
+```
+
+Example usage:
+
+```bash
+$ fluvio profile current
+local
+```
+
+## `fluvio profile delete`
+
+Deletes a profile from your Fluvio configuration (`~/.fluvio/config`). This will
+not delete a cluster (for that, see [`fluvio cluster delete`]), but it will cause
+the Fluvio CLI to no longer be able to interact with that cluster.
+
+```
+fluvio-profile-delete
+Delete the named profile
+
+USAGE:
+    fluvio profile delete <profile name>
+
+FLAGS:
+    -h, --help    Prints help information
+
+ARGS:
+    <profile name>
+```
+
+Example usage:
+
+```bash
+$ fluvio profile delete local
+profile local deleted
+```
+
+If you try to delete the current profile, you will get a warning, because other CLI
+commands depend on having a current profile being set.
+
+```bash
+$ fluvio profile delete local
+profile local deleted
+warning: this removed your current profile, use 'fluvio profile switch' to select a different one
+```
+
+## `fluvio profile delete-cluster`
+
+Deletes cluster connection information from your Fluvio configuration. This will not delete
```

Comment:
This is  confusing, I have no idea how it's different from `delete`

Reply:
Yes, that is an unfortunate consequence of how the config is defined as a separate "profile" and "cluster" portion. I think this should definitely be made nicer to use, but it is a problem in the CLI rather than the docs. I will make an issue for making this more intuitive.

Reply:
https://github.com/infinyon/fluvio/issues/1091